src/listing-list.js

```diff
@@ -86,7 +86,50 @@ export default class ListingList extends React.Component {
     }
     return false
   }
-  render() {
+  renderList() {
+    return location.hash === '#listings'
```

Comment:
You should filter the listings an pass them to `ListingList` in your `renderView` method of `App` to avoid this duplication.